addons/account_asset/migrations/10.0.1.0/pre-migration.py

```diff
@@ -0,0 +1,17 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Tecnativa - Vicent Cubells
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from openupgradelib import openupgrade
+
+
+column_renames = {
+    'account_move': [
+        ('asset_id', None),
+    ],
+}
+
+
+@openupgrade.migrate(use_env=True)
```

Comment:
No need to put `use_env=True`, as it's the default in v10, but it's not so important.